src/olympia/files/tests/test_models.py

```diff
@@ -257,7 +257,10 @@ def test_generate_filename_platform_specific(self):
 
     def test_generate_filename_many_apps(self):
         f = File.objects.get(id=67442)
-        f.version.compatible_apps = (amo.FIREFOX, amo.THUNDERBIRD)
+        f.version.compatible_apps = (amo.THUNDERBIRD, amo.FIREFOX)
+        # After adding sorting for compatible_apps, above becomes
+        # (amo.FIREFOX, amo.THUNDERBIRD) so 'fx+tb' is appended to filename
+        # instead of 'tb+fx'
```

Comment:
It helps understanding the source of the code / comment just by looking at the code instead of the need to use git blame to figure out why things are the way they are.,